code/modules/research/techweb/all_nodes.dm

```diff
@@ -1009,8 +1083,6 @@
 	design_ids = list("kickstart_nanites","factory_nanites","tinker_nanites","offline_nanites")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1000, TECHWEB_POINT_TYPE_NANITES = 2500)
```

Comment:
Protocols here
and it have nanite prise

Reply:
ah, my bad, i was doing a word search on replication protocols